### ISSUE TITLE
Image alt text check is asserting that periods should be at the front of RTL languages

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/i18n-check#70]
### Terms

- [x] I have searched all [open bug reports](https://github.com/activist-org/i18n-check/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [x] I agree to follow activist's [Code of Conduct](.github/CODE_OF_CONDUCT.md)

### Behavior

An issue that we're experiencing in activist with the use of i18n-check right now is that we can't activate the `--alt-text` (`-at`) check. The reason for this is that we have Arabic as a localization, and whenever we turn on the check it starts saying that periods should be added to the front of the texts. We can likely assume that the `--aria-label` (`-al`) check would also fail in the case that there is punctuation at the end of RTL text that should be removed, as it's likely checking the start of the text.

This issue would investigate this bug, find a solution, and update the tests for both of these checks to make sure that the appropriate behavior is being observed :)

**Comment by andrewtavis** [comment]
Happy to get on a call at some point this week if it'd help!

**Comment by andrewtavis** [comment]
Quick check in, @Jag-Marcel :) Do you think you have capacity to work on this? Again totally fine if you'll have some time over the coming month, but if not let us know and maybe @OmkarSathish or @Jolanto would want to jump in there :)

**Comment by Jag-Marcel** [comment]
Hey @andrewtavis 👋 probably won't be working on this any time soon, lots of stuff holding me up now, you can reassign if you want

**Comment by andrewtavis** [comment]
Sounds good, @Jag-Marcel!

@OmkarSathish or @Jolanto, do either of you have interest in picking this up? :)

**Comment by OmkarSathish** [comment]
@andrewtavis sure, either/both of the responsibilities works

**Comment by OmkarSathish** [comment]
Works, I'll start working on it tonight

**Comment by andrewtavis** [comment]
Assigning @Jolanto to reflect the effort coming for the review :) Looking forward to the collaboration!
